src/plugins/chart_expressions/expression_xy/public/helpers/data_layers.tsx

```diff
@@ -55,6 +56,7 @@ type GetSeriesPropsFn = (config: {
   uiState?: PersistedState;
   allYAccessors: Array<string | ExpressionValueVisDimension>;
   singleTable?: boolean;
+  multipleLayersWithSplits: boolean;
```

Comment:
"Splits" reminds me of small multiples. ~Seems like we tend to use use "bucket" or "breakdown" for this kind of thing.~

E.g. https://github.com/elastic/kibana/blob/e327a41af39a2dcb285da936ca2b01076ddb9e4b/src/plugins/chart_expressions/expression_partition_vis/public/utils/filter_helpers.ts#L59-L60

Edit: I see that we use the term "split" in `splitAccessors`. It feels like we could think about some naming standardization.

Reply:
Agree, it's a mess and it will get worse with small multiples if we don't clean it up. I will create a tech debt issue for this